timetypes/go_duration_value_example_test.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package timetypes_test
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-framework-timetypes/timetypes"
+)
+
+type DurationResourceModel struct {
+	Duration timetypes.GoDuration `tfsdk:"duration"`
+}
+
+func ExampleGoDuration_ValueDuration() {
```

Comment:
Looks like golangci-lint is still showing one report:

```
  Error: timetypes/go_duration_value_example_test.go:16:1: tests: ExampleGoDuration_ValueDuration refers to unknown field or method: GoDuration.ValueDuration (govet)
  func ExampleGoDuration_ValueDuration() {
  ^
```

```suggestion
func ExampleGoDuration_ValueGoDuration() {
```

Reply:
Yeah I caught that a few minutes ago and fixed it as well. Thanks